app/models/ssh_execution.rb

```diff
@@ -1,4 +1,7 @@
 class SshExecution
+  class_attribute :executioner
```

Comment:
No that was on purpose :trollface: 

Good catch! Will change in follow up PR :-)